de.metas.business/src/test/java/org/adempiere/impexp/bpartner/IBPartnerFactory.java

```diff
@@ -0,0 +1,110 @@
+package org.adempiere.impexp.bpartner;
+
+import java.util.Properties;
+
+import org.adempiere.ad.trx.api.ITrx;
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.compiere.model.I_I_BPartner;
+
+import lombok.Builder;
+import lombok.Value;
+
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2017 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+/**
+ * Fluent {@link I_I_BPartner} factory
+ *
+ * @author metas-dev <dev@metasfresh.com>
+ */
+@Builder
+@Value
+/* package */class IBPartnerFactory
```

Comment:
couldn't this factory be useful in the "main" code as opposed to "test"?

Reply:
don't think so because the I_I_BPartner records are created by some generic code.